python/ray/rllib/agent.py

```diff
@@ -17,13 +17,15 @@
 
 import tensorflow as tf
 from ray.tune.logger import UnifiedLogger
+from ray.tune.registry import ENV_CREATOR
 from ray.tune.result import TrainingResult
+from ray.tune.trainable import Trainable
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
 
-class Agent(object):
+class Agent(Trainable):
```

Comment:
NIT: In a later PR, this might need some renaming since "ScriptRunner" extends this but technically scriptRunner isn't RLLib related.

Reply:
Yea, tune classes shouldn't inherit from agent, but this change is too big for now.